lisa/operating_system.py

```diff
@@ -1061,6 +1044,15 @@ def _replace_default_entry(self, entry: str) -> None:
         cat = self._node.tools[Cat]
         cat.run("/etc/default/grub")
 
+    @retry(tries=10, delay=5)
```

Comment:
I think it doesn't need to add retry here, unless the `node.execute` raise exception. It rarely happens in LISA. This retry will work with the retry on Debian, so it will retry 10 times together.

Reply:
remove